gillespy2/basic_hybrid_solver.py

```diff
@@ -0,0 +1,127 @@
+from .gillespySolver import GillesPySolver
+import random
+import math
+from scipy.integrate import ode
+
+
+class BasicRootSolver(GillesPySolver):
+    name = "Basic Root Solver"
+
+    @staticmethod
+    def f(pops, species, parameters, reactions):
+        curr_state = {'vol': 1}
+        state_change = []
+        for i, s in enumerate(species):
+            curr_state[s] = pops[i]
+        for p in parameters:
+            curr_state[p] = parameters[p].value
+        for i, r in enumerate(reactions):
+            state_change.append(eval(reactions[r].propensity_function, curr_state))
+        return state_change
+
+    @staticmethod
+    def get_reaction(populations, y0, model, step):
```

Comment:
This method needs to save the population of all "DeterministicSpecies" before it returns.  It also needs to return the  time at which the reaction occurred.